websites/permissions.py

```diff
@@ -237,8 +237,22 @@ def has_object_permission(self, request, view, obj):
 class HasWebsiteCollaborationPermission(BasePermission):
     """Permission to add or remove a website admin/editor"""
 
+    def has_permission(self, request, view):
+        website = get_object_or_404(
+            Website, name=view.kwargs.get("parent_lookup_website", None)
+        )
+        return check_perm(request.user, constants.PERMISSION_COLLABORATE, website)
+
     def has_object_permission(self, request, view, obj):
-        return check_perm(request.user, constants.PERMISSION_COLLABORATE, obj)
+        website = get_object_or_404(
+            Website, name=view.kwargs.get("parent_lookup_website", None)
+        )
+        if request.method not in SAFE_METHODS and (
+            website.owner == obj or is_global_admin(obj)
```

Comment:
I might be missing something but wouldn't we want to return `True` in this expression since it's checking if the user is a owner or admin of the object?

Reply:
`obj` here is the user for which permissions are trying to be changed, not the user making the request, and permissions should never be changed for global admins or website owners.